bom-runtime/pom.xml

```diff
@@ -37,6 +37,22 @@
         <scope>import</scope>
       </dependency>
       <!-- DataStax Java Driver BOM -->
+      <!--
+      Note: we explicitly declare java-driver-core here,
+      and before the driver bom, in order for
+      the exclusions to be taken into account
+      -->
+      <dependency>
+        <groupId>com.datastax.oss</groupId>
+        <artifactId>java-driver-core</artifactId>
+        <version>${datastax-java-driver.version}</version>
+        <exclusions>
+          <exclusion>
+            <groupId>io.dropwizard.metrics</groupId>
```

Comment:
I experimented with existing exclusions and as far as I can tell:

1. Excluding jcl-over-slf4j is IMO a bad idea. This is a bridge from JCL to SLF4J, i.e., it provides an implementation of the JCL API that delegates to SLF4J. Excluding this results in `NoClassDefFoundError`, and the only way to fix it is to include another bridge, namely, commons-logging-jboss-logging. But we are just swapping one bridge for another.
2. Other exclusions: groovy, groovy-json, javapoet, asm-util. AFAICT there is no difference. I suggest that we don't exclude them, unless I'm missing something, but in this case it would be nice to document why we exclude them.

Reply:
This effectively solves #79 .

Reply:
see my comment here:
https://github.com/datastax/cassandra-quarkus/issues/79#issuecomment-646476395

Reply:
As I said in https://github.com/datastax/cassandra-quarkus/pull/77#issuecomment-646616843 , I think we are fine here. We don't need to (and should not) exclude JCL over SLF4J, and any other dependency other than Dropwizard.